docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -52,7 +64,214 @@ to delete this directory and start the node up again. If you are using the defau
 
 :::
 
-## Changing the chain ID
+## Starting a local node (manual configuration)
+
+Manually configuring and starting the Gno blockchain node is a bit more involved than simply initializing it "lazily",
+and involves the following steps:
+
+- generating the node secrets, and configuration
+- generating the `genesis.json`, and populating it
+- starting the node with the generated data
+
```

Comment:
Same as above:
Nit: use a numbered bullet list, start items with a capital letter, and end the sentence with a comma or period.

Reply:
See https://github.com/gnolang/gno/pull/1985#discussion_r1596837191

Reply:
See my reply to the other comment